docs/configuration.md

```diff
@@ -1810,12 +1810,19 @@ def someVeryLongMethodName: Map[String, String] = ???
 
 ### `newlines.beforeOpenParenXxxSite`
 
-If true, a parameter group which doesn't fit on the same line will be formatted
-separately, with a newline just before the opening parenthesis.
+This parameter, if enabled, causes a parameter group which doesn't fit on the
+same line to be formatted separately, with a newline just before the opening
+parenthesis.
 
-Specific formatting depends on the value of `newlines.source` (for `keep`, will
-preserve a newline; for `unfold`, will format every parameter group separately
-unless all fit on a single line, etc.).
+> Since v3.0.0.
+
+Use `null` (default) to disable it. Otherwise, takes the same values as
```

Comment:
```suggestion
Use `classic` (default) to disable it. Otherwise, takes the same values as
```
to be in line with the other names?

Reply:
`classic` cannot be specified explicitly (for `newlines.source` or any other which uses the same type). i specifically made it hidden so that the only way to get the original behaviour is by _not_ specifying anything.

even if `classic` were available, it would mean _enable with classic behaviour`.

the only reason to mention it here is if people want to specify `unfold` but then disable for a subset of files via `fileOverride`.